witness/golang/cmd/witness/main.go

```diff
@@ -44,12 +46,24 @@ func main() {
 		glog.Exitf("Error forming a signer: %v", err)
 	}
 
+	if len(*configFile) == 0 {
+		glog.Exitf("--config_file must not be empty")
```

Comment:
supernit: `glog.Exit()` (no format params)

Reply:
I was just copying the change you made a few lines above :).  But cool, changed them both now.